### PR TITLE
findByStadiumSector method

### PR DESCRIPTION
**Pull request by TolegenKH**
Please check changes and approve it!

### CODE REVIEW

File: src/TicketService.java [hunk 1]
```diff
@@ -1,21 +1,52 @@
...
+    }
+
+    private static Ticket findById(String id, Ticket[] tickets) {
+        for (Ticket ticket : tickets) {
+            if (ticket.getId().equals(id)) {
+                return ticket;
+            }
+        }
+        throw new NoSuchElementException("No ticket found with ID: " + id);
+    }
```
Comment (Alexmansar) [inline src/TicketService.java]: is it a part of second task..?I think its about first))

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@Tolyaspit @AnyaRatusnyak hi, please check the comment)

**Review by Alexmansar — APPROVED** [review]
@Tolyaspit @AnyaRatusnyak Hi, good jod, could be merged as well)

**Review by AnyaRatusnyak — APPROVED** [review]
Good job)

**Pull request merged**